packages/backend/server/tests/auth.e2e.ts

```diff
@@ -52,6 +58,13 @@ test.beforeEach(async t => {
   t.context.app = app;
   t.context.auth = auth;
   t.context.mail = mail;
+
+  // init features
+  const run = module.get(RunCommand);
```

Comment:
we should not involve data migration in testing, just write test cases for current code branches and manually create old data if backwards compatibility test required

tests for data migration is another topic

Reply:
now only run feature config import

Reply:
it still too verbose, we need a convenient way to init everything including db before each tests